Create ui-c8y-1021-22-12-backporting-alarms-filtering-in-widgets-to-y-2025-branch-7884.md (#2802)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-22-12-backporting-alarms-filtering-in-widgets-to-y-2025-branch-7884.md

---
date: ""

> title: Alarms list widget now offers date range filtering ```

Reviewer: ```suggestion
title: Alarms list widget now offers date range filtering
```

> title: Backporting alarms filtering in widgets to y2025 branch (#7884)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Alarm filtering in widgets backported to y2025 branch
```

> title: Backporting alarms filtering in widgets to y2025 branch (#7884)

Reviewer 2: ```suggestion
title: It is now possible to use date filtering for alarms in the alarms page and in the alarm list widget.```

> title: It is now possible to use date filtering for alarms in the alarms page and in the alarm list widget.```

Reviewer 2: ```suggestion
title: Alarms list widget now offers date range filtering ```
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-60796
version: 1021.22.12
---

> Backporting alarms filtering in widgets to y2025 branch (#7884)

Author: The Claude AI suggested to use the following body:
```suggestion
In the y2025 branch, widgets displaying alarms previously showed all alarms without the ability to filter them. This change backports the alarm filtering capability to widgets in the y2025 branch, now allowing users to filter the displayed alarms in widgets based on their preferences. This improvement enhances the user experience and provides better control over the information displayed in alarm widgets, enabling users to focus on relevant alarms in the y2025 branch installations.
```

Reviewer: @eniosultan This is weird. The ticket should simply go into the release/y2025 branch with the same text than the one for develop, not mentioning any backporting. Same way as we always do it for 2024. I wonder why the branch is not correct automatically. Please check with Tristan. Also I don´t know if the version number is different in CD and 2025?

> Backporting alarms filtering in widgets to y2025 branch (#7884)

Reviewer 2: ```suggestion
The “Alarm list” widget previously displayed all alarms without the ability to filter them by date. With this change, users can now filter the displayed alarms in the widget by specifying a date range. This allows users to narrow down the shown alarms to a specific time period they are interested in. Users can select the date range either specifically for the widget or bind the widget to the dashboard time context.
```
